runck

```diff
@@ -77,6 +87,8 @@ launch() {
     chmod 0755 "${tmpfile}"
   fi
 
+  echo ""
```

Comment:
but what is the output on stdout before the file execution?

what if you want to runck a script that writes something to stdout and you want to save that output, e.g.:

```
runck https://mymagicgenerator.com >/tmp/magic
```

this `echo "" ` gets in the way and adds something